client/dashboard/sites/overview/index.tsx

```diff
@@ -28,20 +36,24 @@ import './style.scss';
 
 function SiteOverview() {
 	const { siteSlug } = siteRoute.useParams();
-	const { data } = useQuery( siteQuery( siteSlug ) );
+	const { data: site } = useQuery( siteQuery( siteSlug ) );
+	const { data: mediaStorage } = useQuery( siteMediaStorageQuery( siteSlug ) );
+	const { data: siteMonitorUptime } = useQuery( {
+		...siteMonitorUptimeQuery( siteSlug ),
+		enabled: site?.jetpack && site?.jetpack_modules.includes( 'monitor' ),
+	} );
+	const { data: phpVersion } = useQuery( {
+		...sitePHPVersionQuery( siteSlug ),
+		enabled: site?.options?.is_wpcom_atomic,
+	} );
+	const { data: currentPlan } = useQuery( siteCurrentPlanQuery( siteSlug ) );
+	const { data: primaryDomain } = useQuery( sitePrimaryDomainQuery( siteSlug ) );
+	const { data: engagementStats } = useQuery( siteEngagementStatsQuery( siteSlug ) );
```

Comment:
There's a possibility of moving these calls to each sub-component as well. Not sure yet what I prefer.

Reply:
Yeah. I also think not all of them should block the page load, see [Dashboard v2: performance: lazy load media storage](https://github.com/Automattic/wp-calypso/pull/103549#top) for example.

Reply:
I agree, for example the dependent promises like `phpVersion`; I don't think it should block this component, we can just render it as we gets the value from the server.

Reply:
Yes, I'm going to fix a few. The main thing to look out for is that we don't have a ton of spinners and layout shifts. If it's graceful, then ok.

Reply:
See [Dashboard v2: lazy load php version](https://github.com/Automattic/wp-calypso/pull/103549#top) too